specification/app/resource-manager/Microsoft.App/stable/2024-03-01/AvailableWorkloadProfiles.json

```diff
@@ -100,6 +100,11 @@
               "format": "int32",
               "description": "Memory in GiB."
             },
+            "gpus": {
```

Comment:
This is an internal api but still good practice to introduce new features to preview api-version.

Reply:
`AvailableWorkloadProfiles` api was intended for portal team only.